packages/style-guide/src/ColorPalette.js

```diff
@@ -51,7 +52,6 @@ export const ColorRow = ({
             return render({
               swatch,
               color,
-              key,
```

Comment:
Thanks! I *think* this argument is needed in the `@theme-ui/editor` package – can you keep this in the render prop, please?

Reply:
Here you go.